# Add cert bundle path for Solaris derived operating systems.
# The upstream patch comes from OpenIndiana.
#
#    https://github.com/OpenIndiana/oi-userland/pull/5370/commits/68d615d68511935fa31555eca220847ea5713da7
#
--- emacs-26.3.orig/lisp/net/gnutls.el	2019-01-07 07:26:06.000000000 +0000
+++ emacs-26.3/lisp/net/gnutls.el	2019-11-25 09:34:38.630058874 +0000
@@ -101,6 +101,7 @@
 
 (defcustom gnutls-trustfiles
   '(
+    "/etc/certs/ca-certificates.crt"         ; Solaris
     "/etc/ssl/certs/ca-certificates.crt"     ; Debian, Ubuntu, Gentoo and Arch Linux
     "/etc/pki/tls/certs/ca-bundle.crt"       ; Fedora and RHEL
     "/etc/ssl/ca-bundle.pem"                 ; Suse
